### PR TITLE
C paragraph gallery

### PR DESCRIPTION
**Pull request by imanuelgittens**
Finish paragraph gallery component and add missing dependency to package.json

### CODE REVIEW

File: assets/html/evolution/paragraph-gallery.html [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    </div>
+    <div class="gallery-item__wrapper">
+    <span class="paragraph-gallery__item">
+      <img class="paragraph-gallery__image" src="https://placeimg.com/350/350/tech">
+    </span>
+    </div>
+  </div>
+</article>
+
+<div class="evo_c-markup">
```
Comment (Gruximillian) [inline assets/html/evolution/paragraph-gallery.html]: @imanuelgittens 
Remove this code display section `<div class="evo_c-markup">`, we are not using it anymore.

File: assets/scripts/evolution/paragraph-gallery.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            galleryImage.classList.remove('animate-up');
+          }
+
+
+          let galleryImageRect = galleryImage.getBoundingClientRect();
+          let galleryImageWidth = galleryImageRect.width;
+          let galleryImageHeight = galleryImageRect.height;
+          let galleryImageTop = galleryImageRect.top;
+          let galleryImageLeft = galleryImageRect.left;
+          console.log(galleryImageTop);
```
Comment (Gruximillian) [inline assets/scripts/evolution/paragraph-gallery.js]: @imanuelgittens 

Comment out all console.logs

File: assets/stylesheets/components/evolution/paragraph_gallery/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+##Paragraph Gallery
+
+The paragraph gallery component allows you to  add contextual images to the end of a paragraph.
+The images should relate to the paragraph topics for maximum impact. Paste the following code to the
+end of your paragraph to include a gallery:
+
+    <div class="paragraph-gallery">
```
Comment (Gruximillian) [inline assets/stylesheets/components/evolution/paragraph_gallery/README.md]: @imanuelgittens 

Shouldn't this html markup be wrapped in markdown code block?

**Comment by Gruximillian** [comment]
@imanuelgittens 

Imanuel, please use [naming conventions](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#naming-conventions) for our framework, prefixing classes with `.evo_` and adding BEM identifiers.

**Comment by imanuelgittens** [comment]
I've made these changes @Gruximillian except for the code block part of the README. That already shows up as code since I used indented the block with a tab.

**Comment by Gruximillian** [comment]
@imanuelgittens 
>That already shows up as code since I used indented the block with a tab.

Oh, I haven't opened the file and didn't see that, cool then, sorry for mising it.

**Comment by Gruximillian** [comment]
@imanuelgittens 

Imanuel, please add this component to the list in the [spreadsheet](https://docs.google.com/spreadsheets/d/1nlap-Uv2ScxhchiRDRoOckNk1KckVUQ76D_f7YCzctQ/edit#gid=0). Thanks!

**Pull request merged**